package.json

```diff
@@ -7,13 +7,15 @@
   "main": "src/app.ts",
   "type": "module",
   "scripts": {
-    "dev": "docker compose up",
+    "dev": "docker compose up -d --no-recreate",
```

Comment:
Should we add a script to tear down the env also? (can always do `docker compose down` i guess?)

Reply:
Yeah good thinking - I'll add a `yarn stop` and `yarn fix`